docs/osquery/manage-integration.asciidoc

```diff
@@ -76,12 +77,44 @@ As an example, the following configuration disables two tables.
 +
 ```ts
 {
-   "options":{
-      "disable_tables":"curl,process_envs"
+   "options": {
+      "disable_tables":"file,process_envs"
    }
 }
 ```
 
+[float]
+[[enable-curl-table]]
+==== Enabling the `curl` table
```

Comment:
Not sure why, but I don't see this section in the documentation.

Reply:
I think this was a timing issue. There was a build breakage after this change that is now resolved. You can find this section preview here now: https://kibana_134965.docs-preview.app.elstc.co/guide/en/kibana/master/manage-osquery-integration.html#enable-curl-table